package.json

```diff
@@ -59,7 +59,24 @@
   "pnpm": {
     "overrides": {
       "cookie": "^0.7.2"
-    }
+    },
+    "ignoredBuiltDependencies": [
+      "@bufbuild/buf",
+      "@datadog/native-appsec",
+      "@datadog/native-iast-taint-tracking",
+      "@datadog/native-metrics",
+      "@datadog/pprof",
+      "@swc/core",
+      "cbor-extract",
+      "dd-trace",
+      "esbuild",
+      "protobufjs",
+      "puppeteer"
+    ],
```

Comment:
these may need to be approved, but for now everything works without them being approved from what I can tell.